tox.ini

```diff
@@ -17,9 +17,9 @@ multi_line_output=3
 use_parentheses=True
 
 [flake8]
-max-line-length= 100
+max-line-length=88
 exclude= venv*,.tox,docs,build
-ignore=
+extend-ignore=W503
```

Comment:
Just curious why the `ignore` -> `extend-ignore`. Seems like `ignore` is more appropriate? 🤔

Reply:
I just got it from the black docs. The flake8 docs say:

> The difference to the [--ignore](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore) option is, that this option can be used to selectively add individual codes without overriding the default list entirely.

I honestly can't decipher what that actually means, but it seems like it's less heavy-handed?

Reply:
Ah interesting... glad I asked. We should maybe change this in other libraries where we've implemented `black` as well just for consistency. Just saw it in their docs too  👍

Reply:
yeah, good idea!